### PR TITLE
repo/ci: Shift publishing CI -> github

### PR DESCRIPTION
**Pull request by phlax**
Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by phlax** [comment]
This wont pass CI until it lands as it relies on changes that are not made in a PR - its fairly well tested elsewhere

We need to address infra issues before we can land it - cc @krajshiva

File: ci/do_ci.sh [hunk 1]
```diff
@@ -598,8 +599,10 @@ case $CI_TARGET in
               --//distribution:envoy-contrib-binary=//distribution:custom/envoy-contrib \
               //distribution:packages.tar.gz
         if [[ "${ENVOY_BUILD_ARCH}" == "x86_64" ]]; then
+            echo "Packages created: ${ENVOY_BUILD_DIR}/packages.x64.tar.gz"
             cp -a bazel-bin/distribution/packages.tar.gz "${ENVOY_BUILD_DIR}/packages.x64.tar.gz"
         else
+            echo "Packages created: ${ENVOY_BUILD_DIR}/packages.x64.tar.gz"
```
Comment (phlax) [inline ci/do_ci.sh:605]: remove - same above

Comment (adisuissa) [reply ci/do_ci.sh:605]: probably rename to `packages.arm64.tar.gz`

**Review by adisuissa — COMMENTED** [review]
Overall LGTM, thanks!
I'll assign it to myself to prevent the slack-bot from nagging.
Do you think it makes sense to land this after the next sec-release? (just to reduce the chances of unexpected surprises)

**Comment by phlax** [comment]
>  Do you think it makes sense to land this after the next sec-release? (just to reduce the chances of unexpected surprises)

should be unrelated as this is only change `main` - that said, some work is going to be needed in the sec repo (already, most likely)

main blocker here is there is not enough resources currently allocated

File: ci/do_ci.sh (hunk 1 above)
Comment (phlax) [reply ci/do_ci.sh:605]: i think i prefer it the existing way without the debugging info

**Comment by adisuissa** [comment]
Should this PR be in waiting mode?

**Comment by phlax** [comment]
yeah, until we resolve the infra issue

/wait

**Comment by phlax** [comment]
@adisuissa this one should be gtg - infra issue has been resolved i believe

**Review by ggreenway — APPROVED** [review]

**Pull request merged**